go_lib/dependency/requirements/requirements.go

```diff
@@ -53,6 +54,9 @@ var mreg = regexp.MustCompile(`/modules/([0-9]+-)?(\S+)/requirements`)
 // CheckRequirement run check functions for `key` requirement. Returns true if all checks is passed, false otherwise
 // enabledModules is optional and will filter check-functions if module is disabled
 func CheckRequirement(key, value string, enabledModules ...set.Set) (bool, error) {
+	if os.Getenv("D8_IS_TESTS_ENVIRONMENT") != "" {
+		mreg = regexp.MustCompile(`/modules/([0-9]+-)?(\S+)/hooks`)
```

Comment:
Add comment here to remove when deckhouse-release-controller will be ready

Reply:
done